data/crac-file/crac-impl/src/test/java/com/farao_community/farao/data/crac_impl/threshold/AbsoluteFlowThresholdTest.java

```diff
@@ -69,34 +71,34 @@ public void forbiddenThresholdConstruction() {
         }
         try {
             // forbidden value
-            new AbsoluteFlowThreshold(Unit.AMPERE, Side.LEFT, Direction.BOTH, -500);
+            new AbsoluteFlowThreshold(AMPERE, Side.LEFT, Direction.BOTH, -500);
         } catch (FaraoException e) {
             // should throw
         }
     }
 
     @Test
     public void getMinMaxThreshold() throws SynchronizationException {
-        assertEquals(500.0, absoluteFlowThresholdAmps.getMaxThreshold().orElse(Double.MAX_VALUE), DOUBLE_TOL);
-        assertEquals(1500.0, absoluteFlowThresholdMW.getMaxThreshold().orElse(Double.MAX_VALUE), DOUBLE_TOL);
-        assertEquals(-500.0, absoluteFlowThresholdAmps.getMinThreshold().orElse(Double.MIN_VALUE), DOUBLE_TOL);
-        assertEquals(-1500.0, absoluteFlowThresholdMW.getMinThreshold().orElse(Double.MIN_VALUE), DOUBLE_TOL);
+        assertEquals(500.0, absoluteFlowThresholdAmps.getMaxThreshold(AMPERE).orElse(Double.MAX_VALUE), DOUBLE_TOL);
+        assertEquals(1500.0, absoluteFlowThresholdMW.getMaxThreshold(MEGAWATT).orElse(Double.MAX_VALUE), DOUBLE_TOL);
+        assertEquals(-500.0, absoluteFlowThresholdAmps.getMinThreshold(AMPERE).orElse(Double.MIN_VALUE), DOUBLE_TOL);
+        assertEquals(-1500.0, absoluteFlowThresholdMW.getMinThreshold(MEGAWATT).orElse(Double.MIN_VALUE), DOUBLE_TOL);
```

Comment:
This test can be deleted (it is now redundant with the one below)

Reply:
:ok: